fallout/jobs/job_types/legion/recruitlegionary.dm

```diff
@@ -0,0 +1,27 @@
+/datum/job/f13/legion/recruitlegionary
+	title = "Recruit Legionary"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "The Centurion, Any Decanus."
+	description = "Assist the higher ranks in handling your soldiers, keep them in line and focused on the goals at hand."
```

Comment:
Uh? I assume you didn't intend this description, for recruit legionaries, the lowest rank in the Legion.